analyses/transcriptomic-dimension-reduction/README.md

```diff
@@ -0,0 +1,130 @@
+# Dimension reduction: gene expression data
+
+This module runs dimension reduction—namely PCA, UMAP, and t-SNE—on gene expression data from kallisto and RSEM.
+
+To the run the whole pipeline and produce 4 plots with panels for all three approaches
+
+1. RSEM stranded data
+2. RSEM poly-A data
+3. kallisto stranded data
+4. kallisto poly-A data
+
+with points colored by broad histology, simply run:
+
+```
+./dimension-reduction-plots.sh
+```
+
+## Notes on further applications
```

Comment:
```suggestion
## Notes on further applications
```
Seems like this is the whole README, so feels weird to call it "Notes" maybe just *General Usage* or something?